components/client-bts-wrapper.jsx

```
@@ -0,0 +1,405 @@
+                              <Image
+                                src={section.mainImage.url}
+                                alt={`${section.title} main image`}
+                                fill
+                                className="object-cover"
+                              />
+                            ) : section.videoUrl ? (
+                              <div className="youtube-container">
+                                <div className="preload-overlay"></div>
+                                <iframe
+                                  src={section.videoUrl && section.videoUrl.includes('youtube.com') ?
+                                    `https://www.youtube.com/embed/${section.videoUrl.split('v=')[1]}?autoplay=1&mute=1&loop=1&playlist=${section.videoUrl.split('v=')[1]}&controls=0&showinfo=0&rel=0&modestbranding=1&iv_load_policy=3&enablejsapi=1&playsinline=1` :
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)